events/templates/talks.html

```diff
@@ -0,0 +1,131 @@
+    </svg>Unmute</button>
+  </div>
+  <div id="live-alternative" class="row mb-5">
+    <div class="col-md-6 col-sm-12">
+      <img src="{% static 'img/waiting.gif' %}" />
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <h1>{{live.title}}</h1>
+      <p>
+        Le live de la conférence va bientôt commencer et sera disponible sur cette page.
+        Vous pouvez aussi le regarder sur notre
+        <a href="https://twitch.tv/urlab">chaine Twitch</a>.
+      </p>
+      <p>
+        La conférence devrait commencer aux alentours de {{live.start|time}}.
+      </p>
+      <a href="{% url 'view_event' live.pk %}" class="btn btn-primary">Programme des talks</a>
+    </div>
+
+  </div>
+
+{% if future_events %}
+  <h1>Prochaine{{future_events|pluralize}} conférence{{future_events|pluralize}}</h1>
+  {% for event in future_events %}
+    <div class="card mb-3" >
+      <div class="row no-gutters">
+        <div class="col-md-4">
+          <a href="{% url 'view_event' event.pk %}">
+            {% if event.picture %}
+            <img src="{{event.picture.url}}" class="card-img">
+            {% else %}
+            <img src="{% static 'img/no-talk.png'%}" class="card-img">
+            {% endif %}
+          </a>
+        </div>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title">{{event.title}}</h5>
+            <p class="card-text">{{event.description}}</p>
+            <p class="card-text">
+              <ul>
+                <li>Talk1</li>
+                <li>Talk1</li>
+              </ul>
+            </p>
+            <p class="card-text"><small class="text-muted">{{event.start}}</small></p>
+            <a href="{% url 'view_event' event.pk %}" class="btn btn-primary">Programme</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  {% endfor %}
+{% endif %}
+
+<h1>Conférences organisées par UrLab</h1>
+<div class="row">
+  {% for event in past_events %}
+    <div class="col-lg-3 col-sm-6">
+      <div class="card">
+        <a href="{% url 'view_event' event.pk %}">
+          {% if event.picture %}
+          <img class="card-img-top" src="{{event.picture.url}}" alt="Card image cap">
+          {% else %}
+          <img class="card-img-top" src="{% static 'img/no-talk.png'%}" alt="Card image cap">
+          {% endif %}
+
+        </a>
+        <div class="card-body">
+          <h5 class="card-title">{{event.title}}</h5>
+          <p class="card-text"><small class="text-muted">{{event.start}}</small></p>
+          <a href="{% url 'view_event' event.pk %}" class="btn btn-secondary">Détails</a>
+        </div>
+      </div>
+    </div>
+  {% endfor %}
+</div>
+
+  <script>
+    if (flvjs.isSupported()) {
+      let videoElement = document.getElementById('videoElement');
+      let flvPlayer = flvjs.createPlayer({
+        type: 'flv',
+        url: 'https://stream.okeso.net/live/test.flv'
```

Comment:
L'URL devrait probablement être paramétrable (dans les settings de l'incubator, [constance](https://github.com/jazzband/django-constance), puisqu'on l'utilise https://github.com/UrLab/incubator/blob/main/requirements.in#L9 ?)

Reply:
Smart